src/components/navbar/navbar.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+const navBar = () => {
+    const [categories, setCategories] = useState([])
+
+    // Fetch categories from the api 
+    useEffect(() => {
```

Comment:
I think we have to query the DB instead of fetching externally since our app is already seeded

Reply:
Ok Ill re review, but I wasn't sure how to go about it since the categories in the seed don't match any of the wireframes